### PR TITLE
[Console] Improve the description of command options

### LINKED ISSUE #16648: [console | in-code documentation] Parameters not documented.
Hi,

The parameters $name and $description, in Command::addOption, are not documented in code.

Please see: [Source code](https://github.com/symfony/console/blob/0a4f498e8a2599f269edfd8605b5f03154b7074b/Command/Command.php#L449)

Thx :)

### PR DESCRIPTION
**Pull request by javiereguiluz**
Fixes #16648.

### CODE REVIEW

File: console/input.rst [hunk 1]
```diff
@@ -134,10 +134,17 @@ how many times in a row the message should be printed::
     $this
         // ...
         ->addOption(
+            // this is the name that users must type to pass this option
             'iterations',
+            // this is the shortcut of the option name, which usually is just a letter
```
Comment (94noni) [inline console/input.rst]:
```suggestion
            // this is the optional shortcut of the option name, which usually is just a letter
```

File: console/input.rst [hunk 2]
```diff
@@ -134,10 +134,17 @@ how many times in a row the message should be printed::
     $this
         // ...
         ->addOption(
+            // this is the name that users must type to pass this option
```
Comment (94noni) [inline console/input.rst]: as you mentioned `-I` below, perhaps document here `(e.g. use --iterations=5)` here?

**Comment by javiereguiluz** [comment]
I did the proposed changes. Thanks Antoine!

**Review by 94noni — APPROVED** [review]
Great

**Pull request merged**